packages/agent/src/app.ts

```diff
@@ -665,10 +734,6 @@ export class App {
     }
 
     try {
-      // Stop the agent to prepare for service being restarted
```

Comment:
We no longer want to stop the service here since it happens inside the installer itself now